app/views/devise/registrations/new.html.erb

```diff
@@ -27,9 +28,16 @@
             <div class="col-12">
               <%= f.password_field :password_confirmation, autocomplete: "new-password", label_as_placeholder: true, input_group_class: 'input-group-lg' %>
             </div>
+            <div class="col-12">
```

Comment:
I thought we talked about not putting this on the sign-up page due to it probably causing clutter. Would have to take a look at it in the view. I thought this would be part of `app/views/devise/registrations/edit.html.erb`.